cmd/ci-operator-prowgen/main.go

```diff
@@ -673,7 +683,11 @@ func generateJobs(
 		}
 
 		if element.Cron == nil {
-			presubmits[orgrepo] = append(presubmits[orgrepo], *generatePresubmitForTest(element.As, info, label, podSpec, true, configSpec.CanonicalGoRepository))
+			presubmit := *generatePresubmitForTest(element.As, info, label, podSpec, true, configSpec.CanonicalGoRepository)
+			presubmits[orgrepo] = append(presubmits[orgrepo], presubmit)
+			if migratedPresubmit := deplicatePresubmit(info, &presubmit); migratedPresubmit != nil {
+				presubmits[orgrepo] = append(presubmits[orgrepo], *migratedPresubmit)
```

Comment:
nit: if you need concrete types to append, let's make the functions we call return those instead of pointers